systemjs/react-hello-world/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>SystemJS AMD Modules from CDN Example</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <script type="systemjs-importmap">
+    {
+      "imports": {
+        "react": "https://cdn.jsdelivr.net/npm/react/umd/react.production.min.js",
+        "react-dom": "https://cdn.jsdelivr.net/npm/react-dom/umd/react-dom.production.min.js",
+        "hello-world": "./dist/react-hello-world.js"
```

Comment:
Maybe let's not map the local code rather?

Reply:
Interesting -- I actually have always preferred putting the modules into the import map. I think it's because I like using external import maps where a "module deployment" involves uploading the js file to a server and then updating a "deployed import map" to point to the new version of the module.

I usually do this for both 3p and 1p code.

I'm okay with using a relative url though, and have updated the code to do so.

Reply:
As mentioned if you would prefer to use an import map, then lets use a folder mapping over a file-based one.

Reply:
No, I'm okay with using the relative url.